Sources/AblyChat/AblyCocoaExtensions/Ably+Dependencies.swift

```diff
@@ -9,10 +9,16 @@ import Ably
     extension ARTRealtimeChannels: RealtimeChannelsProtocol, @retroactive @unchecked Sendable {}
 
     extension ARTRealtimeChannel: RealtimeChannelProtocol, @retroactive @unchecked Sendable {}
+
+    extension ARTRealtimePresenceQuery: @retroactive @unchecked Sendable {}
 #else
     extension ARTRealtime: RealtimeClientProtocol, @unchecked Sendable {}
 
     extension ARTRealtimeChannels: RealtimeChannelsProtocol, @unchecked Sendable {}
 
     extension ARTRealtimeChannel: RealtimeChannelProtocol, @unchecked Sendable {}
+
+    extension ARTRealtimePresenceQuery: @unchecked Sendable {}
+
+    extension AsyncStream<Message>.Iterator: @unchecked Sendable {}
```

Comment:
What does this have to do with the SDK? And what's it needed for?

Reply:
I removed it and Xcode doesn't give me any errors anymore. Have no idea, but there is no `AsyncStream<Message>.Iterator` string in the code either.

https://github.com/ably-labs/ably-chat-swift/pull/34/commits/87d6aff0673574d86badce9866bf1c88099aff01